leandra-project/package.json

```diff
@@ -6,7 +6,7 @@
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-redux": "^7.0.3",
-    "react-scripts": "3.0.0",
+    "react-scripts": "^3.0.1",
```

Comment:
Looks like `create-react-app` has an issue in `v3.0.0` with regards to the dependency `babel-jest`. I changed the version to be `^3.0.1` and updated `yarn.lock` accordingly.

https://github.com/facebook/create-react-app/releases/tag/v3.0.1